pkg/api/vlabs/validate.go

```diff
@@ -290,6 +303,13 @@ func (a *Properties) validateNetworkPolicy() error {
 	return nil
 }
 
+func validateNameEmpty(name string, label string) error {
```

Comment:
Saw this, after i merged it. We can do another one to correct the name. There is validateName as well. It needs a broader change.